packages/ia-topnav/src/user-menu.js

```diff
@@ -0,0 +1,39 @@
+import { html } from 'lit-element';
+import DropdownMenu from './dropdown-menu';
+import userMenuCSS from './styles/user-menu';
+
+class UserMenu extends DropdownMenu {
+  static get styles() {
+    return [DropdownMenu.styles, userMenuCSS];
+  }
+
+  static get properties() {
+    const props = {
+      ...DropdownMenu.properties,
+      username: { type: String },
+    };
+    return props;
+  }
+
+  constructor() {
+    super();
+    this.username = 'USERNAME';
```

Comment:
QUESTION: Will this value ever get used anywhere? (Is there a need to initialize to this value?)